### PR TITLE
Feature/clinics view accessibility

### LINKED ISSUE #44: Accessibility for Clinics View
Apply tab indexes and aria labels where needed.

### PR DESCRIPTION
**Pull request by Rallen13**
This needs to be merged AFTER my testing branch since I used that to create this branch.

** resolves #44 **

** Describe the functionality to be tested: **
- adds tab indexes and aria labels to ensure  a pleasant accessible experience

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -1,5 +1,38 @@
 /// <reference types="cypress" />
 
-Cypress.Commands.add("dataCy", (value) => {
+Cypress.Commands.add('dataCy', (value) => {
   return cy.get(`[data-cy=${value}]`);
 });
+
+Cypress.Commands.add('interceptGQL', (url, operation, data, alias) => {
```
Comment (irmakerdem) [inline cypress/support/commands.js:7]: I will need to review this entire file 😿

File: src/Components/ClinicCard/ClinicCard.js [hunk 2]
```diff
@@ -10,36 +10,49 @@ const ClinicCard = ({
   phone,
   services,
   url,
+  ariaStatus,
```
Comment (irmakerdem) [inline src/Components/ClinicCard/ClinicCard.js:13]: whoa 🤯

File: src/Components/ClinicCard/ClinicCard.js [hunk 3]
```diff
@@ -10,36 +10,49 @@ const ClinicCard = ({
...
-          <span className="material-symbols-outlined">location_on</span>{" "}
-          {address}{" "}
+        <p data-cy="clinic-address" className="clinic-address" aria-label={`Location: ${address}.`}>
+          <span className="material-symbols-outlined">location_on</span>{' '}
+          {address}{' '}
         </p>
       )}
       {city && state && zip && (
-        <p data-cy="clinic-city-state-zip" className="clinic-city-state-zip">
+        <p data-cy="clinic-city-state-zip" className="clinic-city-state-zip" aria-label={`${city}, ${state} ${zip}.`}>
```
Comment (irmakerdem) [inline src/Components/ClinicCard/ClinicCard.js:27]: The dynamism of this aria label 🧠

File: src/Components/Footer/Footer.js [hunk 4]
```diff
@@ -8,65 +8,65 @@ const Footer = () => {
         <img src={logo} alt="logo" className="logo-img" data-cy="logo-img"></img>
         <div className="footer-team-container">
           <div className="fe-container">
-            <p data-cy="fe-team">FrontEnd Team:</p>
+            <p data-cy="fe-team" tabIndex={0}>FrontEnd Team:</p>
```
Comment (irmakerdem) [inline src/Components/Footer/Footer.js:11]: Is `tabIndex={0}` the same as `tabIndex="0"`?

**Review by irmakerdem — APPROVED** [review]

File: src/Components/Footer/Footer.js (hunk 4 above)
Comment (Rallen13) [reply src/Components/Footer/Footer.js]: yeah this is how my vscode suggest its written when I start typing. both work.

File: cypress/support/commands.js (hunk 1 above)
Comment (Rallen13) [reply cypress/support/commands.js]: Here is the article where we got this from, does a really awesome job of explaining it and setting it up! 
https://www.jayfreestone.com/writing/stubbing-graphql-cypress/

File: src/Components/ClinicCard/ClinicCard.js (hunk 2 above)
Comment (lkarpins) [reply src/Components/ClinicCard/ClinicCard.js:13]: +1 🤯

**Review by lkarpins — APPROVED** [review]

File: src/Components/ClinicCard/ClinicCard.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-import React from "react";
-import "./ClinicCard.scss";
+import React from 'react';
+import './ClinicCard.scss';
 
 const ClinicCard = ({
```
Comment (catlynbowles) [inline src/Components/ClinicCard/ClinicCard.js:4]: Clinic card machina ! 🙌🏼

File: src/Components/ClinicCard/ClinicCard.js [hunk 6]
```diff
@@ -10,39 +10,62 @@ const ClinicCard = ({
...
+        <p data-cy="clinic-phone" aria-label={`Phone: ${phone}.`}>
           <span className="material-symbols-outlined">phone</span> {phone}
         </p>
       )}
       {services && (
-        <p data-cy="clinic-services">
-          <span className="material-symbols-outlined">medical_services</span>{" "}
+        <p
+          data-cy="clinic-services"
+          aria-label={`Medical services: ${services}.`}
```
Comment (catlynbowles) [inline src/Components/ClinicCard/ClinicCard.js:56]: Excited to see how the screen reader works for this view !

**Review by catlynbowles — APPROVED** [review]

**Pull request merged**